cli/command_snapshot_create.go

```diff
@@ -199,7 +221,7 @@ func snapshotSingleSource(ctx context.Context, rep repo.RepositoryWriter, u *sna
 
 	log(ctx).Debugf("uploading %v using %v previous manifests", sourceInfo, len(previous))
 
-	manifest, err := u.Upload(ctx, fsEntry, policyTree, sourceInfo, previous...)
+	manifest, err := u.Upload(ctx, fsEntry, policyTree, sourceInfo, tags, previous...)
```

Comment:
As Jarek mentioned, the tags do not need to be pass down all the way to the uploader. They simply need to be passed below to `snapshot.SaveSnapshot(....)` in line [254](https://github.com/kastenhq/kopia/pull/138/files#diff-3b29928dc5fae24f6b3dd1434eba47808b18a942853fb6910b8e638e5506db6fR254).